spec.html

```diff
@@ -14659,7 +14664,7 @@ <h2>Syntax</h2>
 
       <emu-clause id="sec-static-semantics-tv-and-trv">
         <h1>Static Semantics: TV and TRV</h1>
-        <p>A template literal component is interpreted as a sequence of Unicode code points. The Template Value (TV) of a literal component is described in terms of String values (SV, <emu-xref href="#sec-literals-string-literals"></emu-xref>) contributed by the various parts of the template literal component. As part of this process, some Unicode code points within the template component are interpreted as having a mathematical value (MV, <emu-xref href="#sec-literals-numeric-literals"></emu-xref>). In determining a TV, escape sequences are replaced by the UTF-16 code unit(s) of the Unicode code point represented by the escape sequence. The Template Raw Value (TRV) is similar to a Template Value with the difference that in TRVs escape sequences are interpreted literally.</p>
+        <p>A template literal component is interpreted by TV and TRV as a value of the String type. In TV, escape sequences are replaced by the UTF-16 code unit(s) of the Unicode code point represented by the escape sequence. TRV is similar to TV with the difference being that in TRV, escape sequences are interpreted as they appear in the literal.</p>
```

Comment:
I am [still not convinced](https://github.com/tc39/ecma262/pull/2435#discussion_r657526179) it makes sense to drop the expansions of TV and TRV, at least not in this PR.

Reply:
I was just in the middle of splitting it out, as we had discussed at the last editor call. See #2451.